readthedocs/projects/models.py

```diff
@@ -403,6 +403,13 @@ class Project(models.Model):
             "Should builds from pull requests be public? <strong>If your repository is public, don't set this to private</strong>."
         ),
     )
+    show_build_overview_in_comment = models.BooleanField(
+        _("Show build overview in a comment"),
+        db_default=False,
+        help_text=_(
+            "Show an overview of the build and files changed in a comment when a pull request is built."
```

Comment:
Do we need to clarify here that this only works with the GitHub app? We should at least link to the docs.

Reply:
Read in the description that this only applies when the GH app is enabled, so that's probably fine for now.